eland/__init__.py

```diff
@@ -14,7 +14,16 @@
 
 # flake8: noqa
 
-from eland._version import *
+from eland._version import (
+    __title__,
+    __description__,
```

Comment:
Looking over this change, I actually much prefer it when the imports are listed out like in the lines above instead of just using the asterisk - it makes it easier to find where things are defined (imho).  @sethmlarson is it common practice to do asterisk imports in __init__.py? I guess it saves some typing, but I've always generally had the feel that asterisk imports in general were frowned upon.

Reply:
Yeah star imports are not optimal, makes it hard for tooling to figure out the true signature of objects being used without searching everywhere. I actually have a flake warning suppressed due to this but I'm saving the fix until we narrow our public API so there's no rework to do. :)